core/src/main/scala/tech/beshu/ror/configuration/loader/distributed/NodesResponse.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.configuration.loader.distributed
+
+import cats.implicits._
+import io.circe.generic.extras.codec.UnwrappedCodec
+import io.circe.syntax._
+import io.circe.{Encoder, Json}
+import shapeless._
+import tech.beshu.ror.accesscontrol.domain.IndexName
+import tech.beshu.ror.configuration.loader.LoadedConfig.FileRecoveredConfig
+import tech.beshu.ror.configuration.loader.distributed.NodesResponse.{ClusterName, NodeError, NodeResponse}
+import tech.beshu.ror.configuration.loader.distributed.Summary.{ConfigurationStatement, Summary}
+import tech.beshu.ror.configuration.loader.{LoadedConfig, Path}
+
+final case class NodesResponse private(clusterName: ClusterName, responses: List[NodeResponse], failures: List[NodeError], summary: Summary)
```

Comment:
I don't think we should send errors. When ROR cannot read or load config, it's not operational (it's UP, but blocking all requests - returns 503).

Reply:
Node can return error because of timeout, or not installed ror plugin. Node could be working, but don't return config.